src/Propel/Generator/Builder/DataModelBuilder.php

```diff
@@ -54,7 +54,7 @@ abstract class DataModelBuilder
     /**
      * An array of warning messages that can be retrieved for display.
      *
-     * @var array string[]
+     * @var array<string>
```

Comment:
mb it should be `list` too?

Reply:
probably, I just fixed up some old ones.